qubesmanager/qube_manager.py

```diff
@@ -746,7 +750,6 @@ def __init__(self, qt_app, qubes_app, dispatcher, _parent=None):
             column = self.qubes_model.columns_indices[col_no]
             action = self.menu_view.addAction(column)
             action.setData(column)
-            action.setCheckable(True)
```

Comment:
This one should stay. I'll fix that when merging.

Reply:
Ohh yes, I noticed that but forgot to fix great :)